internal/params/binds.go

```diff
@@ -23,6 +23,7 @@ var EnvVarsBinds = []struct {
 	{KicsResultsPathKey, KicsResultsPathEnv, "api/kics-results"},
 	{KicsResultsPredicatesPathKey, KicsResultsPredicatesPathEnv, "api/kics-results-predicates"},
 	{BflPathKey, BflPathEnv, "api/bfl"},
+	{PRDecorationPathKey, PRDecorationPathEnv, "api/flow-publisher/pr/github"},
```

Comment:
Should we name the variable specifically for github?

Reply:
ya, makes more sense.